content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -1,64 +1,78 @@
 ---
-title: Install Istio with the Istio CNI plugin
-description: Install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
+title: Install the Istio CNI component
+description: Install and use the Istio CNI component, allowing operators to deploy workloads with lower privilege.
 weight: 70
 aliases:
-    - /docs/setup/kubernetes/install/cni
     - /docs/setup/kubernetes/additional-setup/cni
+    - /docs/setup/additional-setup/cni
 keywords: [cni]
 owner: istio/wg-networking-maintainers
 test: yes
 ---
 
-Follow this guide to install, configure, and use an Istio mesh using the Istio Container Network Interface
-([CNI](https://github.com/containernetworking/cni#cni---the-container-network-interface))
-plugin.
+`istio-cni` is a Istio node agent DaemonSet that runs in your cluster. It is used by both Istio {{< gloss >}}data plane{{< /gloss >}} modes.
+
+`istio-cni` runs in your cluster with elevated privileges, and is used to configure traffic redirection
+for pods in the Istio mesh.
+
+For the {{< gloss >}}sidecar{{< /gloss >}} data plane mode, it is optional, and removes the requirement of running privileged init containers in every pod in the mesh, replacing that model with a single privileged node agent pod on each Kubernetes node.
+
+For the {{< gloss >}}ambient{{< /gloss >}} data plane mode, it is the only supported mechanism, and must be installed.
+
+This guide is focused on using the `istio-cni` component as an optional part of the sidecar data plane mode. Consult [the ambient docs](/docs/ambient/) for information on using the ambient data plane mode.
+
+`istio-cni` makes use of the Container Network Interface ([CNI](https://www.cni.dev/)) plugin API to extend the primary CNI already running in your Kubernetes cluster.
+
+{{< tip >}}
+Note: The `istio-cni` component is not a CNI. Among other things, it installs a chained CNI plugin, which is designed to be layered on top of another, previously-installed primary interface CNI, such as [Calico](https://docs.projectcalico.org), or the cluster CNI used by your cloud provider.
```

Comment:
```suggestion
Note: The `istio-cni` component is _not_ a CNI plugin in the common usage of the term. Among other things, it installs a _chained_ CNI plugin, which is designed to be layered on top of another, previously-installed primary interface CNI, such as [Calico](https://docs.projectcalico.org), or the cluster CNI used by your cloud provider.
```

perhaps lift what I wrote for the glossary and put it here?

Reply:
I had missed/forgot that you had, lemme rework.